bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/WebSocketConnection.java

```diff
@@ -424,7 +440,22 @@ public void onWebSocketClose(int code, @Nullable String reason) {
         @Override
         public void onWebSocketError(@Nullable Throwable error) {
             logger.info("Web Socket error", error);
```

Comment:
I know this is not part of you change, but I wonder if we should reduce this to debug?  Info seem very chatty here.

Reply:
I was thinking of that, too. Maybe we should change that when we fix the underlying connection problem and leave it as-is for now, so we can better track if we are successful there.